diff --git external/skia/Android.mk external/skia/Android.mk
index 2c6575d..230ae3a 100644
--- external/skia/Android.mk
+++ external/skia/Android.mk
@@ -472,9 +472,9 @@ LOCAL_STATIC_LIBRARIES := \
 	libwebp-decode \
 	libwebp-encode
 
-ifeq ($(BOARD_USES_QCOM_HARDWARE),true)
-	LOCAL_WHOLE_STATIC_LIBRARIES += libqc-skia
-endif
+#ifeq ($(BOARD_USES_QCOM_HARDWARE),true)
+#	LOCAL_WHOLE_STATIC_LIBRARIES += libqc-skia
+#endif
 
 
 LOCAL_C_INCLUDES := \
