diff --git external/sqlite/dist/Android.mk external/sqlite/dist/Android.mk
index e61afb7..2197537 100644
--- external/sqlite/dist/Android.mk
+++ external/sqlite/dist/Android.mk
@@ -57,7 +57,7 @@ LOCAL_SHARED_LIBRARIES += liblog \
 LOCAL_WHOLE_STATIC_LIBRARIES := libsqlite3_android
 
 ifeq ($(BOARD_USES_QCOM_HARDWARE),true)
-LOCAL_WHOLE_STATIC_LIBRARIES += libqc-sqlite
+#LOCAL_WHOLE_STATIC_LIBRARIES += libqc-sqlite
 LOCAL_SHARED_LIBRARIES += libcutils
 endif
 
